index.bs

```diff
@@ -1035,7 +1035,7 @@ partial interface MLCommandEncoder {
 </div>
 
 <div class="note">
-Graph initialization stage typically involves a process known as "weight preprocessing" where all the constant inputs to the graph are preprocessed and cached at the operating system level for subsequent graph execution calls. The initializing inputs are typically the constant weight data specified through the {{MLGraphBuilder}}.{{MLGraphBuilder/constant(desc, bufferView)}} method as constant operands during graph construction time.
+Graph initialization stage typically involves a process known as "weight preprocessing" where all the constant inputs to the graph are preprocessed and cached at the operating system level for subsequent graph execution calls. The initializing inputs are typically the constant weight data specified through the <a href="#api-mlgraphbuilder-constant-method">constant()</a> method as constant operands during graph construction time.
```

Comment:
Thanks Dom. In this case I used this form because the function is polymorphic and we have multiple versions. Will need to experiment a bit - and the same problem will be there will all the polymorphic functions.

Reply:
unless I'm mistaken, bikeshed allows to refer to a specific version of a polymorphic function by specifying the parameters

Reply:
But what if I want to refer to the section, not one of the versions? Since here we mean either version.

Reply:
ah, I understand now - then your existing approach is indeed probably the best, sorry for the noise :)

Reply:
With the `{{ x | y }}` notation, I can put in `x` _one_ of the versions and `y` will be shown. But it's not what I want: I'd like to link the section instead. 

So far I figured out 2 choices: the one above using a `<a href=#section-name>` tag, and the one below:
- make a dfn tag in the section title
  `### The <dfn>MLGraphBuilder/constant()</dfn> method  ### {#api-mlgraphbuilder-constant-method}`
- and use `<a>MLGraphBuilder/constant()</a>`

Both are manual linking, and what I used is more natural and versatile.

But if I must use autolinking, I can use this notation, linking to the version first described in the spec:
`{{MLGraphBuilder/constant(descriptor, bufferView)|MLGraphBuilder/constant()}}`

For lack of better, and for making peace, I am going to use that for now.